src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
-
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machine = {truck, bulldozer, excavator};
```

Comment:
The array name `machine` should be plural to reflect that it contains multiple machines. A better name would be `machines`. This enhances the readability and understanding of the code.